### DISCUSSION TITLE
Split multipage tifs into single page tifs wihtout changing compression

### DISCUSSION THREAD

**Discussion by bmacbmac2** [ImageMagick/ImageMagick#5349] [Digital Image Processing]
Hi.  I receive multipage tif images quite often with varying compression formats (fax4, lzw, etc).  Is it possible to extract the single page tif images out of the multipage while maintaining whatever compression was in the multipage?

For example, I'm running:

`magick "c:\temp\Test.tif" -set filename:f "%[t]" +adjoin -scene 1 c:\output\%[filename:f]".%03d`

My original multipage file started as lzw, grayscale compression, however my split files are none, grayscale compression.

I know I could set it to lzw for this file, but when I receive my files I don't know if they are lzw, fax4, etc so I can't manually set this flag.

**Comment by fmw42** [comment]
Add -define tiff:preserve-compression=true before the output

**Comment by bmacbmac2** [comment]
Worked perfectly!!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
